Source/Wikitivity.Agent/WikitivityUploadsAgent.cs

```diff
@@ -2,9 +2,12 @@
 using kCura.Relativity.Client;
 using kCura.Relativity.Client.DTOs;

```

Comment:
Ideally no reference to kCura.Relativity.Client
We may need to make a note, that the packages are currently falsely claiming it as a dependency